e2e/pages/administration/confirm/confirm.spec.js

```diff
@@ -33,6 +36,7 @@ describe('when user is on confirm listing page', () => {
     it('should allow user to reject a file', () => {
       hooks.open('#/administration/confirm/listings');
       confirmPage.rejectListing(rejectListingId1);
+      assert.equal(toast.toastTitle.getText(), 'Success');
```

Comment:
We're switching to `expect`, right?
```suggestion
      expect(toast.toastTitle.getText()).toBe('Success');
```